eds/blocks/call-to-action/call-to-action.js

```diff
@@ -0,0 +1,52 @@
+import {
+  calciteButton,
+  div,
+} from '../../scripts/dom-helpers.js';
+
+export default async function decorate(block) {
+  const ctaItems = [...block.children].map((child) => [...child.children]);
+
+  // get all cta contents
+  const ctaContents = ctaItems.flatMap((ctaItem) => ctaItem);
+
+  ctaContents.forEach((cta) => {
+    // create array out of each cta item
+    const ctaItem = [...cta.children];
+
+    // get p.button-container and change p to div
+    const buttonContainer = ctaItem.find((item) => item.tagName === 'P' && item.classList.contains('button-container'));
+    if (buttonContainer) {
+      const newDiv = div({ class: 'button-container' });
+      while (buttonContainer.firstChild) {
+      // if firstchild is <a> then construct calciteButton
+        if (buttonContainer.firstChild.tagName === 'A') {
+          newDiv.appendChild(calciteButton({
```

Comment:
you could also do: `buttonContainer.replaceWith(calciteButton(...))` or `buttonContainer.replaceChildren(calciteButton(...))`

Reply:
This way you can avoid the later "splice" logic

Reply:
Did you see the latest?